assets/css/style.css

```
@@ -0,0 +1,984 @@
+    --surface: #f6f8f9;
+    --surface-strong: #eef2f4;
+    --border: #dfe5e9;
+    --text: #16212b;
+    --muted: #5a6a78;
+    --link: var(--accent);
+    --accent-soft: color-mix(in srgb, var(--accent) 12%, transparent);
+    --radius: 10px;
+    --maxw: 1080px;
+    --measure: 68ch;
+    --serif:
+        'Iowan Old Style', 'Palatino Linotype', Palatino, 'Book Antiqua',
```

Stylelint (17.14.0)

[error] Expected "Palatino" to be "palatino" (value-keyword-case)

(value-keyword-case)